legacy/ui/legacy/src/main/java/com/fsck/k9/ui/settings/SettingsListFragment.kt

```diff
@@ -131,6 +137,16 @@ class SettingsListFragment : Fragment(), ItemTouchCallback {
                     url = getString(R.string.user_forum_url),
                     icon = Icons.Outlined.Help,
                 )
+
+                if (fundingManager.getFundingType() == FundingType.GOOGLE_PLAY) {
```

Comment:
I think using a `when` statement would make it more obvious how many options there are and what is done in each case.